Jenkinsfile

```diff
@@ -0,0 +1,67 @@
+// List of targets to compile
+def morpheusTargets = [
```

Comment:
Some targets and toolchains are commented as I am not sure if they are officially supported yet.

I removed all references to morpheus, using just `target` now.